media-fonts/open_sans/open_sans-1.0.recipe

```diff
@@ -8,11 +8,30 @@ HOMEPAGE="http://www.google.com/fonts/specimen/Open+Sans"
 # http://www.google.com/fonts#UsePlace:use/Collection:Open+Sans:300italic,400italic,600italic,700italic,800italic,400,300,600,700,800
 # Leave out the checksum here as the file is generated on the fly.
 #SOURCE_URI="http://www.google.com/fonts/download?kit=3hvsV99qyKCBS55e5pvb3ltkqrIMaAZWyLYEoB48lSQ"
-SOURCE_URI="https://dev.gentoo.org/~yngwin/distfiles/open-sans-1.zip"
-CHECKSUM_SHA256="10ebb03bbca8309e622d46b7b94645b64a124cc84c1f7c2e112a3d0b4fad12b1"
+srcGitRev="beaec0837bd21524b57ecb435158f9011fc03999"
+SOURCE_URI="https://github.com/google/fonts/raw/$srcGitRev/apache/opensans/OpenSans-Bold.ttf#noarchive"
+CHECKSUM_SHA256="1b43de2449d39b65ff6f63315d4afda585f72fbbec2e3d9a56f59de6c75149d3"
+SOURCE_URI_2="https://github.com/google/fonts/raw/$srcGitRev/apache/opensans/OpenSans-BoldItalic.ttf#noarchive"
+CHECKSUM_SHA256_2="3575d2afaaad69970380237a5d6357b6db241f53b77607482eaf9f299b8c07ec"
+SOURCE_URI_3="https://github.com/google/fonts/raw/$srcGitRev/apache/opensans/OpenSans-ExtraBold.ttf#noarchive"
+CHECKSUM_SHA256_3="395f150240d43dff8baea6586baf5665337de57b8204a501fbd6148b2fe165b7"
+SOURCE_URI_4="https://github.com/google/fonts/raw/$srcGitRev/apache/opensans/OpenSans-ExtraBoldItalic.ttf#noarchive"
+CHECKSUM_SHA256_4="d5de39bcdd010089d9db8dd1aebaefaf9e691bf9a49282ff43e1d1869b417892"
+SOURCE_URI_5="https://github.com/google/fonts/raw/$srcGitRev/apache/opensans/OpenSans-Italic.ttf#noarchive"
+CHECKSUM_SHA256_5="6cb918a707a06c4f98221d09344af4b98c9cb6184b13309a579caf0418d5eb74"
+SOURCE_URI_6="https://github.com/google/fonts/raw/$srcGitRev/apache/opensans/OpenSans-Light.ttf#noarchive"
+CHECKSUM_SHA256_6="1c8d3cc6810ecd3623ebff7d2c3db1a44024260c5ae662f8166d69b9425828ed"
+SOURCE_URI_7="https://github.com/google/fonts/raw/$srcGitRev/apache/opensans/OpenSans-LightItalic.ttf#noarchive"
+CHECKSUM_SHA256_7="fda70df85987b394ff384b899703bc0e55ac7bdba94d06f47462e155cf0c0350"
+SOURCE_URI_8="https://github.com/google/fonts/raw/$srcGitRev/apache/opensans/OpenSans-Regular.ttf#noarchive"
+CHECKSUM_SHA256_8="13c03e22a633919beb2847c58c8285fb8a735ee97097d7c48fd403f8294b05f8"
+SOURCE_URI_9="https://github.com/google/fonts/raw/$srcGitRev/apache/opensans/OpenSans-SemiBold.ttf#noarchive"
+CHECKSUM_SHA256_9="b4c2050b25d3d296d5cf58589ca00816dc72df42262c2f629d5c6a984a161aa4"
+SOURCE_URI_10="https://github.com/google/fonts/raw/$srcGitRev/apache/opensans/OpenSans-SemiBoldItalic.ttf#noarchive"
+CHECKSUM_SHA256_10="a8f2af8e79f46686b1cfcfb3a1fd53e94e88308d7c6ee7f85c733f4796fcc3a0"
 LICENSE="Apache v2"
 COPYRIGHT="2010-2011 Google Corporation"
-REVISION="1"
+REVISION="2"
 
 ARCHITECTURES="?any"
```

Comment:
good catch, didn't notice it :) will try to see if I can check it later on x86_64 (don't have x86 install anymore), but as no compiler etc is dealing with it I think it should be ok (tested it with FontDemo on x86_gcc2) ...